### PR TITLE
Update to es6

### LINKED ISSUE #3: update to es6

### PR DESCRIPTION
**Pull request by btnwtn**
Fixes #3 

I've also run prettier on all the JS files.

### CODE REVIEW

File: src/server/bin/www [hunk 1]
```diff
@@ -4,37 +4,37 @@
...
 
 /**
  * Listen on provided port, on all network interfaces.
  */
 
 server.listen(port);
-server.on('error', onError);
-server.on('listening', onListening);
+server.on("error", onError);
+server.on("listening", onListening);
```
Comment (cprass) [inline src/server/bin/www]: Isn't `onListening` in the TDZ here? I think that'll throw

Comment (btnwtn) [reply src/server/bin/www]: TDZ?

Comment (cprass) [reply src/server/bin/www]: Temporal Dead Zone. `const` isn't hoisted like `var` is.

Comment (btnwtn) [reply src/server/bin/www]: Yup you were totally right, guess I'm used to webpack hoisting stuff 👍 

Fixed.

**Pull request merged**